src/Weather.js

```diff
@@ -0,0 +1,8 @@
+class Weather {
+
+    getWeather() {
+        return Math.random() * 10 >= 2 ? `sunny` : `stormy`;
+    }
+}
+
```

Comment:
getWeather method - As weather class has no properties I would have made this method a static method